index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="css/styles.css">
+  <title> youtube-clone</title>
+</head>
+<body>
+  <div id="main">
+    <section class="navbar">
+      <form class="search-form" action="index.html" method="post">
+        <input id="search" type="text" name="search" value="" placeholder="Search">
+      </form>
+    </section>
+    <section class="video-section">
+      <iframe width="750" height="415"
+              src="https://www.youtube.com/embed/ZnuwB35GYMY"
+              frameborder="0" allow="accelerometer; autoplay;
```

Comment:
Hi, guys the `frameborder` attribute is no longer supported by HTML. it may be better to not specify a frameborder, and use CSS to apply borders instead if you need it. 

Please address in all occurrences 😄 .